packages/k6/.env

```diff
@@ -1,4 +1,4 @@
-TEST_DURATION=12h
+TEST_DURATION=2m
```

Comment:
Also flagging just for visibility in case this needs to be updated

Reply:
Yeah I'll revert this one